account_vat_period_end_statement/README.rst

```diff
@@ -152,7 +152,7 @@ Contributors
 * Simone Rubino <simone.rubino@agilebg.com>
 * Giacomo Grasso <giacomo.grasso.82@gmail.com>
 * Lara Baggio <http://linkgroup.it/>
-
+* Gianmarco Conte <gconte@dinamicheaziendali.it>
```

Comment:
ti devi aggiungere nel file https://github.com/OCA/l10n-italy/blob/12.0/account_vat_period_end_statement/readme/CONTRIBUTORS.rst
Se ne occupa OCA Bot (rif. https://github.com/OCA/oca-github-bot#on-push-to-main-branches) di aggiornare il README file. In caso contrario rischi che venga sovrascritto in un futuro aggiornamento del README.

Reply:
Grazie per la dritta Alex, corretto.